katello/katello-backup

```diff
@@ -1,94 +1,88 @@
-#!/bin/bash
+#!/usr/bin/env ruby
 
-usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
-}
+require 'optparse'
+require 'fileutils'
 
-if [ "$#" -ne 1 ]; then
-  echo "ERROR: Please specify an export directory"
-  usage
-  exit 1
-fi
+optparse = OptionParser.new do |opts|
+  opts.banner = "Usage: katello-backup /path/to/dir [options]\neg: $ katello-backup /tmp/katello-backup"
 
-# Backup dir
-umask 0027
-BDIR=$1
-mkdir -p $BDIR
-chgrp postgres $BDIR
-chmod 770 $BDIR
-cd $BDIR
-echo "Creating backup folder $BDIR"
-echo ""
+  opts.on_tail("-h", "--help", "Show this help message and exit") do
```

Comment:
`OptionParser` handles help output for you, you shouldn't need to specify it here.